net.sf.eclipsecs-feature/pom.xml

```diff
@@ -3,7 +3,7 @@
     <parent>
         <artifactId>net.sf.eclipsecs.parent</artifactId>
         <groupId>net.sf.eclipsecs</groupId>
-        <version>10.5.0-SNAPSHOT</version>
+        <version>10.6.0-SNAPSHOT</version>
```

Comment:
I believe this was handled by the release script: https://github.com/checkstyle/eclipse-cs/blob/master/release.sh#L35
You probably will want to update it.